nativelib/src/main/scala/scala/scalanative/meta/LinktimeInfo.scala

```diff
@@ -60,6 +60,11 @@ object LinktimeInfo {
   )
   def contendedPaddingWidth: Int = resolved
 
+  @resolvedAtLinktime(
+    "scala.scalanative.meta.linktimeinfo.runtimeVersion"
+  )
+  def runtimeVersion: String = resolved
```

Comment:
Maybe `version` or `scalaNativeVersion` would be more suitable?

Reply:
I think that current wording is good enough.